packages/checksum/src/cksum.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'cksum' {
```

Comment:
I presume we're doing this because there is no published typing for the cksum module?

Reply:
Correct.